config/rewrites.d/support.rackspace.com.json

```diff
@@ -1238,6 +1238,13 @@
         "to": "/how-to/upgrade-a-cloud-databases-instance-from-mysql-56-to-mysql-57/",
         "rewrite": false,
         "status": 301
+      },
+      {
+        "description": "Retire {Email sorting issues on iOS and Android after an email migration}",
+        "from": "^\\/how-to\\/{email-sorting-issues-on-ios-and-android-after-an-email-migration/}(.*)$",
```

Comment:
You'll need to take the brackets out of the `from` URL here, so:

`"from": "^\\/how-to\\/email-sorting-issues-on-ios-and-android-after-an-email-migration(.*)$",`

The `(.*)$` at the end acts as a wild card, so it'll pick up the URL whether there's a trailing slash or not, or if there are any GA marketing tags in the URL. That sort of thing.